.azure/pipelines/ci.yml

```diff
@@ -727,6 +740,17 @@ stages:
         agentOs: Windows
         timeoutInMinutes: 240
         steps:
+        - task: Cache@2
+          inputs:
+            key: 'pw | "$(Agent.OS)" | eng/versions.props'
+            restoreKeys: |
+               pw | "$(Agent.OS)"
+            path: $(PLAYWRIGHT_BROWSERS_PATH)
+            cacheHitVar: CACHE_PLAYWRIGHT_HIT
+          displayName: Cache playwright browsers
+        - script: npm i -D playwright
+          condition: ne(variables['CACHE_PLAYWRIGHT_HIT'], 'true')
```

Comment:
Why does the Helix test job need Playwright❔ Suggest leaving this job alone

Reply:
Well the quarantined version of helix will need it, and eventually the real one too

Reply:
Don't we do Playwright installation on the Helix agents❔ I could understand this _if_ the installations were uploaded from the build agents.

Reply:
Right now it does it both on the helix machines but also as part of local builds/tests since the developer workflow is a thing too, generally it doesn't look like our builds are factored in a way that makes it easy to optionally install dependencies.  I had to do some magic with conditional compilation for that to work on helix, so its possible but currently we aren't setup to easily stop installing playwright on builds